### ISSUE TITLE
Cannot create/delete multiple dashboards resources at once

### ISSUE THREAD

**Issue by teddylear** [grafana/terraform-provider-grafana#64]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
Terraform version v0.11.11
Grafana provider v1.4.0

### Affected Resource(s)
grafana_dashboard

### Expected Behavior
What should have happened?
Updated the two json files used by template files, which are used by two different grafana_dashboard resources.  Should have deleted existing dashboards and created with new dashboards with updated json. 

### Actual Behavior
What actually happened?
Only one dashboard was deleted and created, the other failed to delete and returned an '500 Internal server error' 

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. Deploy two dashboard resources through terraform
2. Update json of these template files in some way
3. terraform apply and get 500 back for one of the dashboards.

**Comment by tomweston** [comment]
Does anyone know of a fix for this yet?

**Comment by BensamV** [comment]
Any update on this??

**Comment by circa10a** [comment]
Using Grafana v6.7.3

I was able to create multiple dashboards at once and didn't receive an error from the provider although I did notice this in the logs

```log
lvl=info msg="Database locked, sleeping then retrying" logger=sqlstore error="database is locked"`
```

I attempted with both the embedded sqllite database and a postgres database. This is due to grafana locking tables and isn't a problem with the provider. Concurrent resource modifications are built into terraform, so modifying the provider to work around this isn't the right solution. One way around this would be to force terraform to only update one thing at a time instead of updating resources in parallel. This can be achieved with the `-parallelism=1` argument. Example:

```shell
terraform apply -auto-approve -parallelism=1
```
